Python/test/swap.py

```diff
@@ -0,0 +1,142 @@
+"""
```

Comment:
Just a note: for the future, please don't feel obliged to replicate in Python tests that we already have in C++.  I'm happy with those ones.  Here, I'd write tests for stuff which is specific to Python, if any.

Reply:
Ok, understood. Writing these tests gives me additional comfort that I exposed the functions correctly. But I'll keep that in mind going forward, thanks.